README.md

```diff
@@ -30,12 +32,34 @@ export default {
 new NpmAutoInstallWebpackPlugin(options)
 ```
 
-### options.autoInstall
+### options.install
 
 Auto install missing dependencies with `npm install` command.
 
 * Default: `true`
 
+### options.strategy
+
+Custom function for getting modules installed.
+
+* Default: null
+
+```js
+// webpack.config.babel.js
+import NpmAutoInstallWebpackPlugin from "npm-auto-install-webpack-plugin";
+
+export default {
+  plugins: [
+    new NpmAutoInstallWebpackPlugin({
+      install: true,
+      strategy (modules) {
+        someTechnique(['yarn', 'install', ...modules]);
```

Comment:
Good catch. But I'd suggesting returning a promise instead (so that user will know this function should return a promise to indicate installation is done).

```
strategy (modules) {
  return promisfiedSpawn(['yarn', 'install', ...modules])
}
```